frontend/public/components/secret.jsx

```diff
@@ -58,7 +58,25 @@ const SecretDetails = ({obj: secret}) => {
 const SecretsList = props => <List {...props} Header={SecretHeader} Row={SecretRow} />;
 SecretsList.displayName = 'SecretsList';
 
-const SecretsPage = props => <ListPage ListComponent={SecretsList} canCreate={true} {...props} />;
+const secretType = secret => secret.type;
+
+const filters = [{
+  type: 'secret-type',
+  selected: ['kubernetes.io/service-account-token', 'kubernetes.io/dockercfg', 'kubernetes.io/dockerconfigjson', 'kubernetes.io/basic-auth', 'kubernetes.io/ssh-auth', 'kubernetes.io/tls', 'Opaque'],
+  reducer: secretType,
+  items: [
+    {id: 'kubernetes.io/service-account-token', title: 'service-account-token'},
+    {id: 'kubernetes.io/dockercfg', title: 'dockercfg'},
+    {id: 'kubernetes.io/dockerconfigjson', title: 'dockerconfigjson'},
+    {id: 'kubernetes.io/basic-auth', title: 'basic-auth'},
+    {id: 'kubernetes.io/ssh-auth', title: 'ssh-auth'},
+    {id: 'kubernetes.io/tls', title: 'tls'},
+    {id: 'Opaque', title: 'Opaque'}
```

Comment:
Is there a reason for this ordering? I might just make it alphabetical if there's no obvious order.

Reply:
Originally I wanted to have the k8s types first but sort them alphabetically makes sense :+1: